util.py

```diff
@@ -173,17 +192,47 @@ def kill_robot(self, robot):
 
 class BeltTile(Tile):
     def __init__(self, direction, path, properties):
-        self.crossroads = properties[0]["value"]
-        self.belt_direction = properties[1]["value"]
-        self.move_count = properties[2]["value"]
+        self.direction_out = self.transform_direction(properties[0]["value"])
+        self.express = properties[1]["value"]
         super().__init__(direction, path, properties)
 
-    def move_robot(self, state):
-        # TO DO!
-
-        # 1) Express belts move 1 space
-        # 2) Express belts and normal belts move 1 space
-        pass
+    def transform_direction(self, direction_integer):
+        """
+        Method to transform the integer taken from json properties
+        to valid Direction or Rotation class instance for later processing.
+        """
+        if direction_integer == 0:
+            return Direction.N
+        if direction_integer == 90:
+            return Rotation.RIGHT
+        if direction_integer == -90:
+            return Rotation.LEFT
+        if direction_integer == 180:
+            return Rotation.U_TURN
+
+    def check_belts(self, express_belts):
+        # Only express belts
+        if self.express is express_belts:
+            return True
+        # All belts
+        elif express_belts is False:
+            return True
+        else:
+            return False
+
+    def rotate_robot_on_belt(self, robot, direction):
```

Comment:
Zde trochu podobná situace: i s vysvětlivky nevím, jaký případ se zde ošetřuje. V komentáři je zmíněn směr sever, ale v kódu otočení o 180. Můžeš dovysětlit?

Reply:
Souhlasím, že to bude chtít asi jinak popsat. Ten speciální případ je pro křižovatku, kterou máme v obrázcích pojmenovanou jako `conveyor_belt_2_cross_t.png` + varianta s jednoduchým pásem:
![obrazek](https://user-images.githubusercontent.com/38920177/54680466-360ba600-4b0a-11e9-94f1-959039fafd42.png)
Takže by se asi teoreticky dalo napsat T-cross nebo něco takového. 
A ta jako jediná má atribut `direction_out` rovný `Rotation.U_TURN`. Když je tato dlaždice otočená na sever, tak robot ji opouští směrem na jih. To jsem se tam snažila říct. K otočení směru o 180° nedochází. 
Jak dochází k otáčení směru na této křižovatce asi nedokážu tady popsat, chce to nakreslit :)